features/step_definitions/setup_steps.rb

```diff
@@ -4,6 +4,10 @@
 require "json"
 require "test/unit"
 
+@buyer_id = nil
+@published_brief = nil
```

Comment:
I don't think this is correct. In Ruby `@` means an instance variable. As far as I can tell this means instance variables on the module.